introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I'm brand spankin' new to git, but Code School is doing a fantastic job getting me up to speed.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.